utils/consts.go

```diff
@@ -543,6 +543,11 @@ const (
 	MetaSetVersions = "*set_versions"
 )
 
+// MetaFilterIndexesAPIs
+const (
+	ComputeFilterIndexes = "ApierV1.ComputeFilterIndexes"
```

Comment:
the name needs to contain full method, not only last part, please see the other methods